PSICompile/ILCodeGen.cs

```diff
@@ -121,7 +130,7 @@ public class ILCodeGen : Visitor {
    public override void Visit (NUnary u) {
       u.Expr.Accept (this);
       string op = u.Op.Kind.ToString ().ToLower ();
-      op = op switch { "sub" => "neg", _ => op };
+      op = op switch { "sub" => "neg", "not" => "ldc.i4.1 xor", _ => op };

```

Comment:
As Arvind pointed out, there should be only one op code per line. In this case, you have two op code in one line (ldc.i4.1 xor)